src/api/CortexClient.ts

```diff
@@ -285,14 +285,11 @@ export class CortexClient implements CortexApi {
       headers: { 'Content-Type': 'application/json' },
     });
 
-    const responseBody = await response.json();
```

Comment:
this fails for the updated sync, which does not return any JSON back

Reply:
Ah but I see that [there is only 1 place we call this](https://cs.github.com/cortexapps/backstage-backend-plugin?q=post). I think this is OK, but a potential land mine for the future. At lest the Typing is clear -- we can always adjust if/when we need.